### DISCUSSION TITLE
creating endnotes in Word docx format

### DISCUSSION THREAD

**Discussion by donboyd5** [quarto-dev/quarto-cli#675] [Q&A] [answered]
Hi. I hope this is the right place to suggest/ask about this.

When writing documents for non-technical audiences, sometimes it is nice to put technical matter in endnotes rather than footnotes. MS Word easily allows an endnotes section at the end of the document (or, with more trouble, in a section that is not the last section -- e.g., before an appendix). 

It would be great if endnotes were allowed in quarto qmd files for the docx format. As far as I can tell, it is only possible to use footnotes, which appear at the bottom of a page and can be distracting to a non-technical reader.

Would much appreciate it if someone can tell me whether there currently is a way to create endnotes in quarto for the docx format. If not, I would like to put it on a wish list of new features.

Thanks.

**Comment by jjallaire** [comment, marked as answer]
This is possible for some formats using the `reference-location` option. From the Pandoc documentation:

```
--reference-location=block|section|document

Specify whether footnotes (and references, if reference-links is set) are placed
at the end of the current (top-level) block, the current section, or the document. 
The default is document. Currently this option only affects the markdown, muse, 
html, epub, slidy, s5, slideous, dzslides, and revealjs writers.
```

Ultimately we'll need this to implemented natively in Pandoc. It's probably worth posting an issue there (https://github.com/jgm/pandoc/issues) to see whether this is something that would be considered.

**Comment by donboyd5** [comment]
Many thanks. I believe GitHub will cross reference this, but FWIW it appears to be an ongoing discussion on pandoc and I have added my hope that it can be implemented, [here](https://github.com/jgm/pandoc/issues/1425) and [here](https://github.com/jgm/pandoc/issues/4041). There is a related quite old not-passing PR [here](https://github.com/jgm/pandoc/pull/4042).
